RELEASE.md

```diff
@@ -4,19 +4,16 @@
 
 The release process consists of these steps:
   1. Create a pull request with:
-     * Change the version in [`build.gradle`](build.gradle) to match the release version e.g. `0.20.0`
      * Add an entry to [`CHANGELOG`](CHANGELOG.md) with changes since the last release
-  1. Commit your `CHANGELOG` changes
+     * add a new section to the CHANGELOG with that version as `(unreleased)` and a bullet point `- Nothing yet`
+     * bump the version in `build.gradle` to `major.minor.(patch+1)-SNAPSHOT` eg. `0.20.1-SNAPSHOT` - notice the `SNAPSHOT`!
```

Comment:
@yurishkuro would this satisfy your snapshot requirements? 

This way we have a snapshot version like before we just set it in advance. The only difference is if somebody checkouts a tag the and does `gradlew build` it builds snapshot instead of final.